src/networking.c

```diff
@@ -2855,8 +2863,8 @@ int checkClientOutputBufferLimits(client *c) {
         } else {
             time_t elapsed = server.unixtime - c->obuf_soft_limit_reached_time;
 
-            if (elapsed <=
-                server.client_obuf_limits[class].soft_limit_seconds) {
+            if (!server.client_obuf_limits[class].soft_limit_seconds ||
```

Comment:
If we do that, I think we also should supplement some comments in redis.conf. At before, if `server.client_obuf_limits[class].soft_limit_seconds` is 0, it will reach output buffer soft limit at next call this function. But now, we will disable soft limit if this configuration is 0, right?

Reply:
yes, this change of behavior (if we choose to go with it) needs to be clearly documented and also mentioned in the release notes.
p.s. i don't think 0 means the next call to this function, it means at least 1 second (but it's far from being accurate since it looks at unix time in seconds rather than a delta in milliseconds).
i don't suppose anyone is really relying on this.

Reply:
Oh, my bad, next second rather than next call